embassy-stm32/src/usart/mod.rs

```diff
@@ -95,10 +95,10 @@ pub(crate) mod sealed {
         fn af_num(&self) -> u8;
     }
 
-    #[cfg(any(dma, dmamux))]
+    #[cfg(any(bdma, dma, dmamux))]
```

Comment:
isn't this cfg "always on" now that we have all?

Reply:
Maybe? But I don't have enough confidence that there isn't some weird chip lacking any DMA... No harm, no foul leaving it for now, yeah?